src/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx

```diff
@@ -324,7 +324,7 @@ export default function WalletBreakdownView({
           </>
         )}
         <PoweredBy>
-          Powered by <img src={coinGeckoIcon} title="CoinGecko" alt="CoinGecko" />
+          {translateRaw('POWERED_BY')} <img src={coinGeckoIcon} title="CoinGecko" alt="CoinGecko" />
```

Comment:
This can use the PoweredByText component, yeh? Same question for the Powered By Zapper component in `@features/Defizap/defizaplogo`